### ISSUE TITLE
argument parser in upload.py

### ISSUE THREAD

**Issue by rneher** [nextstrain/fauna#83]
the parser is currently instantiated in `vdb/upload.py`. repeated imports from `upload` will, therefore, use the exact same parser and addition of arguments in other files generate argument clashes. code following this line 

https://github.com/nextstrain/fauna/blob/master/vdb/upload.py#L9

should be wrapped into a function `get_parser()` or similar.

**Comment by 4rund3v** [comment]
I think i can work on this , if this is still relavent.

**Comment by jstoja** [comment]
I don't think this is relevant anymore as it is described since it has been wrapped already.
Although, reading through `vdb/download.py`, It seems that it's doing a lot of parsing of the arguments. This probably could be extracted to a `vdb/parser.py`.

**Comment by tjGhani** [comment]
@trvrb I believe this issue can be closed since the issue seems resolved and the parser is wrapped in a function, as originally suggested.
